seismic/inventory/inventory_util.py

```diff
@@ -11,6 +11,7 @@
 from obspy import read_inventory
 
 from seismic.inventory.iris_query import set_text_encoding, form_response_request_url
+from ordered_set import OrderedSet as set
```

Comment:
I would prefer this be reverted, as this module does not need an ordered set.

Reply:
Are you certain that results across multiple runs would be identical with Py3? Having that consistency with results helps with debugging/diffing etc.

Reply:
Same comment as above.